index.html

```diff
@@ -26,8 +26,125 @@ <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
           <li><a href="#"><img src="image/Vector-(1).png" alt="Email icon" class="space"></a></li>
           <li><a href="#"><img src="image/Vector-(2).png" alt="Social media" class="space"></a></li>
         </ul>
-      </section>
-  </div>
-</main>
+     </section>
+    </div>
+  
+    <div class="cards">
+      <div class="card">
+        <img src="image/Snapshoot-Portfolio.png" alt="A photo of a website page">
+        <div class="card-content">
+          <div class="card-heading">Tonic</div>
+          <ul class="tool1">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>        
+          <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
```

Comment:
- [ ] Please wrap this text in a `p` tag instead, in that case, it will be more meaningful than, considering following best practice, please. let's go and fix that.